code/web/src/modules/user/StyleSurvey.js

```diff
@@ -14,25 +15,73 @@ import { white, grey, grey2, grey3 } from "../../ui/common/colors";
 import { APP_URL } from "../../setup/config/env";
 import userRoutes from "../../setup/routes/user";
 
-const StyleSurvey = () => {
+const StyleSurvey = (props) => {
+  const styleDescriptions = {
+    classy:
+      "A Classy style Personality will always be elegantly and appropriately dressed for any occasion and always be well turned out.  Your style is simple, refined and somewhat restrained!",
+    nature:
+      "Feeling comfortable in your clothes is most important to you and your easygoing nature tends towards more casual outfits. Simple lines and designs are your preferred choices over anything too detailed or fussy. In keeping with this carefree attitude you tend to buy only easy, wash and wear garments. You prefer fabrics such as denim, cotton jersey and lightweight knits. Your jewellery choices reflect your minimal look and you tend to wear basic and durable accessories with footwear also focussed on comfort. ",
+    sporty:
+      "A Sporty style includes all forms of dressing with ease: from activewear to outdoorsy, sportswear to menswear. It’s about comfort; comfort from your clothes and with your body.",
+    punk: "this thing",
+    artsy:
+      "You’re way of dressing is innovative and individualistic and you aren’t overly influenced by current trends or traditional rules. You use your clothing choices to reflect your personality, and put together unique and interesting outfits with items purchased from varied sources. Different colours, textures and prints fill your wardrobe and your jewellery collection is bold while your footwear and accessories are usually statement making. ",
+  };
+
   return (
     <section>
       <Helmet>
         <title>My Style Preferences</title>
       </Helmet>
-      <Grid style={{ backgroundColor: grey }}>
-        <GridCell style={{ padding: "2em", textAlign: "center" }}>
-          <H3 font="secondary">My Style Preferences</H3>
-
-          <p style={{ marginTop: "1em", color: grey2 }}>
-            Please complete the survey below.
-          </p>
-        </GridCell>
-      </Grid>
-      <GridCell style={{ textAlign: "center", backgroundColor: grey }}>
-        {/* <p style={{ marginBottom: '1em', color: grey2 }}>Like what you see?</p> */}
-        <StyleSurveyBox />
-      </GridCell>
+      {!props.user.stylePreference ? (
```

Comment:
Added logic for the props - if a survey preference isn't set, load a survey for the user